### PR TITLE
Event stream formatter

### PR DESCRIPTION
**Pull request by l3pp4rd**
This formatter is a step forward to cucumber unified formatter, see cucumber/cucumber-pretty-formatter#2 based on event [spec 0.1.0](https://github.com/cucumber/cucumber-pretty-formatter/blob/refactor/README.md#events)

This branch is still a work in progress.

### CODE REVIEW

File: features/lang.feature [hunk 1]
```diff
@@ -8,10 +8,11 @@ Savybė: užkrauti savybes
   Scenarijus: savybių užkrovimas iš aplanko
     Duota savybių aplankas "features"
     Kai aš išskaitau savybes
-    Tada aš turėčiau turėti 7 savybių failus:
+    Tada aš turėčiau turėti 8 savybių failus:
```
Comment (kairotavares) [inline features/lang.feature]: why this ?

Comment (kairotavares) [inline features/lang.feature]: It is required???

Comment (l3pp4rd) [inline features/lang.feature]: because events.feature file was added, and test suite tests itself

**Comment by l3pp4rd** [comment]
was merged in mainly except pipe through cunicorn, should be worked on elsewhere

**Pull request closed without merging**